README.md

```diff
@@ -854,6 +855,35 @@ datetime
 
 ```
 
+
+### Social Dominance
+
+Returns the % of the social dominance a given project has over time in a given social channel.
+
+[Premium metric](#premium-metrics)
+
+```python
+san.get(
+    "social_dominance/bitcoin",
+    from_date="2019-04-08",
+    to_date="2019-04-13",
+    interval="1d",
+    source="ALL"
```

Comment:
Can we list all possible `source` values?

Reply:
If you go to https://api.santiment.net/graphiql and search in `Docs` in the top right you can find them: 
<img width="652" alt="image" src="https://user-images.githubusercontent.com/6518376/60816821-3015ff80-a1a3-11e9-9b1b-55bb658fdc88.png">